src/cljs/athens/page.cljs

```diff
@@ -70,14 +70,37 @@
        [:div {:style {:margin-left 20}}
         [render-blocks (:block/uid @node)]]])))
 
+(defn title [title]
```

Comment:
a bit dangerous to give the argument the same name as the function (shadowing)

Reply:
Agreed, will change this